samcli/lib/telemetry/metric.py

```diff
@@ -138,14 +140,21 @@ def wrapped(*args, **kwargs):
 
         try:
             ctx = Context.get_current_context()
+            try:
+                template_dict = ctx.template_dict
+            except AttributeError:
+                LOG.debug("Ignoring CDK project check as template is not provided in context.")
+                template_dict = {}
+            project_type = ProjectTypes.CDK.value if is_cdk_project(template_dict) else ProjectTypes.CFN.value
+            metric_specific_attributes = get_all_experimental_statues() if ctx.experimental else {}
+            metric_specific_attributes["projectType"] = project_type
```

Comment:
should the property name `projecttype` ?? .. this value will be part of the JSON value of the column `metricSpecificAttributes` in the metrics data, and I think our queries is sensitive if the value is an uppercase or lowercase.

Reply:
our database is case-insensitive so it doesn't really matter.